### DISCUSSION TITLE
Precipitation_sum broken in August?

### DISCUSSION THREAD

**Discussion by BCPdoc** [open-meteo/open-meteo#411] [Q&A] [answered]
I think that the forecast API is reporting bad data for precipitation_sum starting about August 2nd. For example, if I ask for this:

https://archive-api.open-meteo.com/v1/archive?latitude=41.3693&longitude=-73.9663&start_date=2023-07-30&end_date=2023-08-09&daily=precipitation_sum,rain_sum,snowfall_sum&temperature_unit=fahrenheit&windspeed_unit=mph&precipitation_unit=inch&timezone=auto

It will return null / NaN results for August 2nd, 2023 and on:

time,precipitation_sum (inch),rain_sum (inch),snowfall_sum (inch)
2023-07-30,0.000,0.000,0.000
2023-07-31,0.000,0.000,0.000
2023-08-01,0.000,0.000,0.000
2023-08-02,NaN,NaN,NaN
2023-08-03,NaN,NaN,NaN
2023-08-04,NaN,NaN,NaN
2023-08-05,NaN,NaN,NaN
2023-08-06,NaN,NaN,NaN
2023-08-07,NaN,NaN,NaN
2023-08-08,NaN,NaN,NaN
2023-08-09,NaN,NaN,NaN

I can't actually find *any* locations that give good results. Same null values for rain_sum and snow_sum. It looks to me like a universal problem starting around August 3rd at 1:00 GMT.

Am I missing something? Or is something broken?

**Comment by patrick-zippenfenig** [comment, marked as answer]
Hi,

the `archive-api` endpoint is for historical data. It has a delay of 5-7 days which you are seeing here. For forecast data, just use the endpoint `api.open-meteo.com/v1/forecast`

E.g. https://api.open-meteo.com/v1/forecast?latitude=41.3693&longitude=-73.9663&start_date=2023-07-30&end_date=2023-08-09&daily=precipitation_sum,rain_sum,snowfall_sum&temperature_unit=fahrenheit&windspeed_unit=mph&precipitation_unit=inch&timezone=auto
